components/org.wso2.carbon.identity.oauth.endpoint/src/test/resources/testng.xml

```diff
@@ -28,6 +28,7 @@
             <class name="org.wso2.carbon.identity.oauth.endpoint.authz.OAuth2AuthzEndpointTest" />
             <class name="org.wso2.carbon.identity.oauth.endpoint.introspection.IntrospectionResponseBuilderTest" />
             <class name="org.wso2.carbon.identity.oauth.endpoint.user.OpenIDConnectUserEndpointTest" />
+            <class name="org.wso2.carbon.identity.oauth.endpoint.device.DeviceEndpointTest"></class>
```

Comment:
Can change to <class name="org.wso2.carbon.identity.oauth.endpoint.device.DeviceEndpointTest"/>